spec/std/http/headers_spec.cr

```diff
@@ -160,6 +160,21 @@ describe HTTP::Headers do
     headers.includes_word?("foo", "baz").should be_true
   end
 
+  it "doesn't match word with comma separated value, partial match" do
+    headers = HTTP::Headers{"foo" => "bar, bazo"}
+    headers.includes_word?("foo", "baz").should be_false
+  end
+
+  it "matches word with comma separated value, partial match (array)" do
+    headers = HTTP::Headers{"foo" => ["bar", "bazo, baz"]}
```

Comment:
These examples values are pretty confusing, using the same order is easier to read:
```suggestion
    headers = HTTP::Headers{"foo" => ["foo", "baz, bazo"]}
```